code/src/App.js

```diff
@@ -1,9 +1,101 @@
-import React from 'react'
+import React, { useState } from "react";
+import "./app.css";
+const drinkGroups = ["coffee", "tea", "water"];
 
+/* Statehooks */
 export const App = () => {
+  const [name, setName] = useState("");
+  const [fika, setFika] = useState("");
+  const [drinkGroup, setDrinkgroup] = useState();
+  const [place, setPlace] = useState("");
+  const [summary, setSummary] = useState(false);
+
+  /*   Show the summary when clicking submit */
+  const handleSubmit = event => {
+    event.preventDefault();
+    setSummary(true);
+  };
+
   return (
-    <div>
-      Find me in src/app.js!
+    <div className="formContainer">
+      <form onSubmit={handleSubmit}>
+        <section className="questionCard">
+          <label>
+            <p>Hey you, what's your name?</p>
+            {/* controlled form */}
+            <input
+              className="nameField"
+              type="text"
+              onChange={event =>
+                setName(event.target.value)
+              } /* invoke setName with event.target.value */
+              value={name} /* reflect current state */
+              required
+            />
+          </label>
+        </section>
+        {/* RADIOBUTTONS */}
+        <section className="questionCard">
+          <p>What drink do you prefer to your "fika":</p>
+          {drinkGroups.map(group => (
+            <div className="radioDiv">
+              <label key={group}>
+                <input
+                  type="radio"
+                  value={group}
+                  onChange={event => setDrinkgroup(event.target.value)}
+                  checked={drinkGroup === group}
+                />
+                {group}
+              </label>
+            </div>
+          ))}
+        </section>
+
+        <section className="questionCard">
+          <label>
+            <p>What do you prefer to eat when you "fikar"?</p>
+            <input
+              className="fikaField"
+              type="text"
+              onChange={event => setFika(event.target.value)}
+              value={fika}
+              required
+            />
+          </label>
+        </section>
+
+        <section className="questionCard">
+          <label>
+            {/* DROPDOWN */}
+            <p>Where do you want to "fika"?</p>
+            {/* use select instead of form on dropdown */}
+            <select
+              className="formDropdown"
+              onChange={event => setPlace(event.target.value)}
+              value={place}
+            >
+              <option value="">Choose one!</option> {/* Label, no value */}
```

Comment:
I guess you don't want this one to be selected :)
```suggestion
              <option value="" disabled>Choose one!</option> {/* Label, no value */}
```